templates/image/show.html

```diff
@@ -142,6 +142,9 @@ <h4 class="shake-details-title">In These Shakes</h4>
         <li class="facebook">
           <a target="_blank" href="https://www.facebook.com/sharer/sharer.php?u={{url_escape('https://mltshp.com/p/' + str(sharedfile.share_key))}}&amp;quote={{url_escape(sharedfile.get_title())}}&amp;display=popup" onclick="window.open('https://www.facebook.com/sharer/sharer.php?u={{url_escape('https://mltshp.com/p/' + str(sharedfile.share_key) )}}&amp;quote={{url_escape(sharedfile.get_title())}}&amp;display=popup','sharer','toolbar=0,status=0,width=626,height=436');return false;"><img src="{{ static_url("images/fb_icon.png") }}" width="21" height="21" alt="Post to Facebook"></a>
         </li>
+        <li class="tumblr">
+          <a target="_blank" href="https://tumblr.com/widgets/share/tool?canonicalUrl={{url_escape('https://mltshp.com/p/' + str(sharedfile.share_key))}}&amp;title={{url_escape(sharedfile.get_title())}}&amp;posttype={% if sourcefile.type == 'image' %}photo&amp;content={{url_escape('https://mltshp-cdn.com/r/' + str(sharedfile.share_key))}}{% end %}{% if sourcefile.mp4_flag == 1 %}video&amp;content={{url_escape('https://mltshp-cdn.com/r/' + str(sharedfile.share_key))}}.mp4{% end %}{% sourcefile.webm_flag == 1 %}video&amp;content={{url_escape('https://mltshp-cdn.com/r/' + str(sharedfile.share_key))}}.webm{% end %}&amp;tags=mltshp" onclick="window.open(this.href,'sharer','toolbar=0,status=0,width=626,height=436');return false;"><div class="inner"><span class="text"> Post </span></div></a>
```

Comment:
Missing a `if` for the `{% sourcefile.webm_flag == 1 %}` block here. That's causing the test failure.